app/models.py

```diff
@@ -252,15 +254,17 @@ def is_validate(self, method, src_ip=''):
                     if LDAP_GROUP_SECURITY_ENABLED:
                         try:
                             if LDAP_TYPE == 'ldap':
-                                if (self.ldap_search(searchFilter, LDAP_ADMIN_GROUP)):
+                                groupSearchFilter = "(&({0}={1}){2})".format(LDAP_FILTER_GROUPNAME, ldap_username, LDAP_FILTER_GROUP)
```

Comment:
I don't understand this line. What is the relationship between `LDAP_FILTER_GROUPNAME` and `ldap_username`?
Also, the `ldap_filter_group` and `ldap_filter_groupname` are required in FILTERS section, why its back-end code is under `LDAP_GROUP_SECURITY_ENABLED` which is optional?

Reply:
```logfile
[2019-07-18 13:59:14,215] [DEBUG] | Ldap searchFilter (&(uid=warf)(&(objectclass=posixAccount)(!(nsaccountlock=True))))
[2019-07-18 13:59:14,326] [DEBUG] | Ldap search result: [[('uid=warf,ou=people,dc=example,dc=com', (...)
[2019-07-18 13:59:14,374] [INFO] | groupSearchFilter is (&(member=uid=warf,ou=people,dc=example,dc=com)(objectclass=groupOfNames))
[2019-07-18 13:59:14,420] [INFO] | User warf is part of the "cn=dnsadmins,ou=groups,dc=example,dc=com" group that allows operator access to PowerDNS-Admin
```
So we are looking only in groups (`(objectclass=groupOfNames)`)

Btw. this PR is working  thanks @johnwarburton for patch

Waiting for merge